scripts/tslint/noIncrementDecrementRule.ts

```diff
@@ -0,0 +1,37 @@
+import * as Lint from "tslint/lib/lint";
+import * as ts from "typescript";
+
+
+export class Rule extends Lint.Rules.AbstractRule {
+    public static POSTFIX_FAILURE_STRING = "Don't use '++' or '--' postfix operators outside statements, for loops, or element access expressions.";

```

Comment:
You need to fix the error message to no longer claim element access expressions are an exception.